packager/mpd/base/xml/xml_node.cc

```diff
@@ -583,9 +589,12 @@ bool RepresentationXmlNode::AddAudioChannelInfo(const AudioInfo& audio_info) {
       // Audio Compression (AC-4) Standard; Part 2: Immersive and personalized
       // audio G.3.1.
       const uint32_t ac4_channel_mask =
-        base::HostToNet32(codec_data.channel_mask() << 8);
+          absl::big_endian::FromHost32(codec_data.channel_mask() << 8);
+      absl::Span<const uint8_t> bytes(
+          reinterpret_cast<const uint8_t*>(&ac4_channel_mask),
+          sizeof(ac4_channel_mask));
       audio_channel_config_value =
-        base::HexEncode(&ac4_channel_mask, sizeof(ac4_channel_mask) - 1);
+          absl::StrFormat("%02X%02X%02X", bytes[0], bytes[1], bytes[2]);
```

Comment:
likewise here for the `uint32_t`, perhaps I should the same approach for both AC3 and AC4 though.

Reply:
Same as above, I think.  Swap and print/format/hexify without needing Span or individual bytes.

Also, it took me a while to see the "- 1" on the original.  Please add a comment about us needing just 3 bytes.

Reply:
done